small bug fixes && ping delay for websites added

iconquestion opened the pull request:
small bug fixes

iconquestion committed:
Added PAC proxy detection and operation

Now sduchecktool supports detecting if a system PAC proxy is set and turn off PAC setting!

iconquestion committed:
Merge branch 'master' of https://github.com/ICONQUESTION/SduNetCheckTool

iconquestion committed:
Update RegUtil.cs

iconquestion committed:
Bug Fixes

bug fixes

iconquestion committed:
Merge branch 'master' of https://github.com/ICONQUESTION/SduNetCheckTool

iconquestion committed:
Merge branch 'master' of https://github.com/ICONQUESTION/SduNetCheckTool

iconquestion committed:
update

update

iconquestion committed:
Merge branch 'master' of https://github.com/ICONQUESTION/SduNetCheckTool

Forgot-Dream commented on SduNetCheckTool.Core/Tests/SystemProxyTest.cs:
```diff
@@ -11,6 +11,9 @@ public class SystemProxyTest : ITest
         {
             var data = new List<string>();
             var result = TestResult.Failed;
+            var result1 = TestResult.Failed;
```
建议把原来的`result`变量删除，并且推荐命名`result1`->`commonProxyEnabledResult`,`result2`->`pacProxyEnabledResult`

iconquestion replied:
太讲究了哈哈哈 ok 等下回去改
其实可以稍微简单粗暴一点，用bool表示result就可以了，无需enum自定义类型

iconquestion replied:
这个程序是个不错的规范代码学习模版

Forgot-Dream replied:
> 太讲究了哈哈哈 ok 等下回去改 其实可以稍微简单粗暴一点，用bool表示result就可以了，无需enum自定义类型

以后新增结果可能方便点，比如说我还要加个`可能有问题`的枚举之类的（

iconquestion committed:
update ping delay detection for websites

update ping delay detection for websites.
Most of my time was given to fundamental knowledge!!!

iconquestion committed:
update

update
